packages/themes/src/g80.js

```diff
@@ -0,0 +1,315 @@
+  green40,
+  green50,
+  yellow30,
+  red30,
+  red40,
+  red60,
+  red80,
+  purple40,
+
+  // Constants
+  white,
+
+  // Tools
+  rgba,
+} from '@carbon/colors';
+
+// New color tokens
+// TO-DO: remove fallback color when v11 is released and assign carbon colors to new tokens
+export const background = gray80;
+export const layer = gray70;
+export const layerAccent = gray60;
+export const field = gray70;
+export const backgroundInverse = gray10;
+export const backgroundBrand = blue60;
+export const interactive = blue40;
+
+export const borderSubtle = gray60;
+export const borderStrong = gray40;
+export const borderInverse = gray10;
+export const borderInteractive = blue50;
+
+export const textPrimary = gray10;
+export const textSecondary = gray30;
+export const textPlaceholder = gray60;
+export const textHelper = gray40;
+export const textError = red30;
+export const textOnColor = white;
+export const textInverse = gray100;
+
+export const linkPrimary = blue40;
+export const linkSecondary = blue30;
+export const linkVisited = purple40;
+export const linkInverse = blue60;
+
+export const iconPrimary = gray10;
+export const iconSecondary = gray30;
+export const iconOnColor = white;
+export const iconInverse = gray100;
+
+export const supportError = red40;
+export const supportSuccess = green40;
+export const supportWarning = yellow30;
+export const supportInfo = blue50;
+export const supportErrorInverse = red60;
+export const supportSuccessInverse = green50;
+export const supportWarningInverse = yellow30;
+export const supportInfoInverse = blue60;
+
+export const overlay = rgba(gray100, 0.7);
+export const toggleOff = gray50;
+
+export const buttonPrimary = blue60;
+export const buttonSecondary = gray60;
+export const buttonTertiary = white;
+export const buttonDangerPrimary = red60;
+export const buttonDangerSecondary = red40;
+export const buttonSeparator = gray100;
+
+export const backgroundActive = gray70;
+export const layerActive = gray60;
+
+export const buttonDangerActive = red80;
+export const buttonPrimaryActive = blue80;
+export const buttonSecondaryActive = gray70;
+export const buttonTertiaryActive = gray30;
+
+export const focus = white;
+export const focusInset = gray100;
+export const focusInverse = blue60;
+export const highlight = blue70;
+
+export const backgroundHover = adjustLightness(background, +6);
+export const layerHover = adjustLightness(layer, +7);
+export const fieldHover = adjustLightness(field, +7);
+export const backgroundInverseHover = adjustLightness(backgroundInverse, -5);
+
+export const linkPrimaryHover = blue30;
+
+export const buttonDangerHover = adjustLightness(buttonDangerPrimary, -8);
+export const buttonPrimaryHover = adjustLightness(buttonPrimary, -8);
+export const buttonSecondaryHover = adjustLightness(buttonSecondary, -7);
+export const buttonTertiaryHover = white;
+
+export const backgroundSelected = gray70;
+export const backgroundSelectedHover = adjustLightness(layerSelected, -7);
```

Comment:
`backgroundSelected` not `layerSelected`

Reply:
This is my bad, its typed wrong in the notion doc too.

Reply:
Oops actually it should be `+7` as well. I may have misspoke earlier i think (can't find that previous comment)